peachjam/templates/peachjam/_document_content_sidebar.html

```diff
@@ -10,6 +10,18 @@
   <div class="navigation__inner">
     <nav>
       <div class="nav nav-tabs" id="navigation-tab" role="tablist">
+        {% if document.doc_type == "judgment" and document.case_summary %}
+          <button class="nav-link"
+                  id="doc-summary-tab"
+                  data-bs-toggle="tab"
+                  data-bs-target="#doc-summary"
+                  type="button"
+                  role="tab"
+                  aria-controls="nav-sum"
```

Comment:
this must be the id of the summary tab-pane (ie `aria-controls="doc-summary"`)

the `aria-controls` of the other tabs should also match their tab pane ids